hieradata_aws/production.yaml

```diff
@@ -152,6 +152,7 @@ govuk::apps::support_api::zendesk_anonymous_ticket_email: 'zd-api-public@digital
 govuk::apps::support_api::pp_data_url: 'https://www.performance.service.gov.uk'
 govuk::apps::support_api::aws_s3_bucket_name: 'govuk-production-support-api-csvs'
 govuk::apps::travel_advice_publisher::enable_email_alerts: true
+govuk::apps::whitehall::monitoring_ip: "%{hiera('hosts::production::management::hosts.monitoring-1.ip')}"
```

Comment:
In AWS we won't have this available, I think it's better to leave it out to be explicit. If the Rake task which checks consistency between AWS and Carrenza is failing, you can add this option as an exception.